lib/claim/fees-claimant.js

```diff
@@ -2,8 +2,10 @@
 
 const TokenHolderRevenueFundContractsEnsemble = require('./token-holder-revenue-fund-contracts-ensemble');
 const NestedError = require('../nested-error');
+const {ethers: {utils: {formatUnits}}} = require('ethers');
```

Comment:
Pedantic comment: Can also be written more tersely
`const {formatUnits} = require('ethers').utils;`

Reply:
I reckon that in that case you can just as well go all the way to
```
const formatUnits = require('ethers').utils.formatUnits;
```

I reckon my preference for the initial one is the use of destructuring multiple variables on the same call to `require()`.